sample/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
```

Comment:
I'm attempting manual installation so we don't have to downgrade. Although if it's always going to install these dependencies it hurts build time pretty badly.